Makefile

```diff
@@ -175,11 +175,6 @@ graplctl: ## Build graplctl and install it to the project root
 build-ux: ## Build website assets
 	cd src/js/engagement_view && yarn install && yarn build
 
-# Create the dist directory if necessary; don't need to document this
-# formally for `make help`, though.
-dist:
-	mkdir dist
-
```

Comment:
If you're going to delete this, you should also delete the dependency of the `ux-tarball` target, too.

Reply:
Ah, yes. Good catch.